src/content/LandingPage/LandingPage.js

```diff
@@ -1,7 +1,103 @@
 import React from 'react';
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  Button,
+  Tabs,
+  Tab,
+} from 'carbon-components-react';
+
+const props = {
+  tabs: {
+    selected: 0,
+    triggerHref: '#',
+    role: 'navigation',
+  },
+  tab: {
+    href: '#',
+    role: 'presentation',
+    tabIndex: 0,
+  },
+};
 
 const LandingPage = () => {
-  return <div>LANDING PAGE</div>;
+  return (
+    <div className="bx--grid bx--grid--full-width landing-page">
+      <div className="bx--row landing-page__banner">
+        <div className="bx--col-lg-16">
+          <Breadcrumb noTrailingSlash aria-label="Page navigation">
+            <BreadcrumbItem>
+              <a href="/">Getting started</a>
+            </BreadcrumbItem>
+          </Breadcrumb>
+          <h1 className="landing-page__heading">
+            Design &amp; build with Carbon
+          </h1>
+        </div>
+      </div>
+      <div className="bx--col-lg-16">1</div>
```

Comment:
This <div> needs to be removed - it's affecting your layout. See your preview: https://deploy-preview-2049--carbon-tutorial.netlify.com/#/